### ISSUE TITLE
TRENT: Add box for specifying unused numbers

### ISSUE THREAD

**Issue by sebastiannielsen** [defuse/defuse.ca#16]
A suggestion:
Add a box to allow to specify a comma-separated list of unused numbers.
These numbers will simply be skipped during random generation.

For example, if you run a contest with 10 numbers.
8 entrants enter and number 8 and 6 remain unused.

Then you could specify 1-10.
and then "6,8" as unused numbers.

The random generation could technically work like an 1-8 lottery, but if any number higher than 5 is chosen (in the above example), then 1 is added to the random output, and if the resulting number is higher than 7, add 1 again. (in the 7 case, 1 is added because its higher than 5, which puts it higher than 7, which causes 1 to be added again).
Basically, the translation table would be:
1->1
2->2
3->3
4->4
5->5
6->7
7->9
8->10
